src/pages/ProjectStart/ProjectStart.js

```diff
@@ -1,7 +1,136 @@
 import React from 'react';
+import styled from 'styled-components/macro';
+import { css } from 'styled-components';
+import ProjectStartCard from './Components/ProjectStartCard';
 
 function ProjectStart() {
-  return <></>;
+  const number = Math.floor(Math.random() * Object.keys(TITLE).length);
+  return (
+    <ProjectStartWrapper>
+      <header>
+        <h1>{TITLE[number]}</h1>
+        <button>
+          <i className="fas fa-paper-plane" />
+          프로젝트 등록하기
+        </button>
+      </header>
+      <ProjectStartNav>
+        <NavBtn selected>
+          <div />
+          프로젝트 개요
+        </NavBtn>
+        <NavBtn>
+          <div />
+          펀딩 및 선물 구성
+        </NavBtn>
+      </ProjectStartNav>
+      <section>
+        {CARD_TITLE.map(({ id, cardTitle }) => (
+          <ProjectStartCard key={id} cardTitle={cardTitle} />
+        ))}
+      </section>
+    </ProjectStartWrapper>
+  );
 }
 
+const ProjectStartWrapper = styled.div`
+  ${({ theme }) => theme.flexColumnSet()};
+  margin-top: 2rem;
+
+  & > header {
+    ${({ theme }) => theme.flexColumnSet()};
+    padding: 4em 0;
+  }
+
+  & > header > h1 {
+    margin-bottom: 4rem;
+    color: ${({ theme }) => theme.colors.tertiaryGray};
+    font-size: 2rem;
+  }
+
+  & > header > button {
+    padding: 0.5em 1em;
+    color: ${({ theme }) => theme.colors.red};
+    font-weight: 700;
+    border: 3px solid ${({ theme }) => theme.colors.red};
+    border-radius: 5px;
+
+    & i {
+      margin-right: 10px;
+    }
+  }
+
+  & > section {
+    ${({ theme }) => theme.flexColumnSet()}
+    padding-bottom: 3em;
+    width: 100%;
+    background-color: ${({ theme }) => theme.colors.bgGray};
+  }
+`;
+
+const ProjectStartNav = styled.nav`
+  ${({ theme }) => theme.flexSet()}
+  margin-top: 1rem;
+  width: 100%;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.bgGray};
+`;
+
+const NavBtn = styled.button`
+  ${({ theme }) => theme.flexColumnSet()}
+  padding: 1em 3em;
+  background-color: ${props =>
+    props.selected
+      ? css`
+          ${({ theme }) => theme.colors.bgGray}
+        `
+      : 'white'};
```

Comment:
```suggestion
  background-color: ${({ theme, selected }) =>
    selected ? theme.colors.bgGray : 'white' };
```
이렇게도 가능할 것 같은데 한번 시도해보시겠어요?

Reply:
잘 돼요! 감사합니다~